recipes/s1reader/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "s1reader" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/isce-framework/s1-reader/archive/v{{ version }}.tar.gz
+  sha256: 676f502665d4af29d7774c3c67d607d34496d8c9b5f1cf548f2ba8ccad234721
+
+build:
+  entry_points:
+    - s1_info = s1reader.s1_info:main
+  noarch: python
+  script:
+    - {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - setuptools >=42.0
+    - pip >=21.3
+  run:
+    - python >=3.8
+    - numpy >=1.20
+    - gdal >=3.0
+    - isce3 >=0.13
+    - packaging >=21.0
+    - requests >=2.0
+    - shapely >=1.8
+    - lxml >=4.8
+    - importlib-resources # Next release will remove this dependency
+
+test:
+  imports:
+    - s1reader
+  commands:
+    - pip check
+    - s1_info --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/isce-framework/s1-reader
+  summary: A Sentinel-1 reader for ISCE3
+  license: Apache-2.0
+  license_file: LICENSE
+  dev_url: https://github.com/isce-framework/s1-reader
+
+extra:
+  recipe-maintainers:
+    - scottstanie
+    - seongsujeong
+    - LiangJYu
+    - gshiroma
+    - vbrancat
+    - rtburns-jpl
```

Comment:
We need all the maintainers listed here to agree.  We are missing:
    - LiangJYu
    - gshiroma
    - vbrancat

Reply:
Sorry about that. I've kept the maintainers who have agreed.